specs/014-guest-contributions-privileges/quickstart.md

```
@@ -0,0 +1,573 @@
+## Running Tests
+
+### Unit Tests
+
+```bash
+# Run all unit tests for authorization service
+pnpm test:ci src/services/api/whiteboard/whiteboard.service.authorization.spec.ts
+```
+
+**Expected Output**:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)